x/sequencers/client/cli/tx.go

```diff
@@ -0,0 +1,125 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/dymensionxyz/dymension-rdk/x/sequencers/types"
+)
+
+// GetTxCmd returns the transaction commands for the module.
+func GetTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                        "sequencer",
+		Short:                      fmt.Sprintf("%s transactions subcommands", "sequencer"),
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	cmd.AddCommand(
+		UnsafeImportConsensusKeyCmd(),
+		NewCreateCmd(),
+		NewUpdateCmd(),
+	)
+
+	return cmd
+}
+
+func NewCreateCmd() *cobra.Command {
+	short := "Create a sequencer object, to claim rewards etc."
+	long := strings.TrimSpace(short +
+		`Requires signature from consensus address public key. Specify consensus key in keyring uid.
+Operator addr should be bech32 encoded. You may supply a different reward addr optionally.`)
+
+	cmd := &cobra.Command{
+		Use:     "create-sequencer [key name] {reward addr}",
+		Example: "create-sequencer fooCons --reward-addr ethm1cv7qcksr7cyxv9wgjn3tpxd74n2pffryq7ujw4  --from foo",
+		Args:    cobra.ExactArgs(1),
+		Short:   short,
+		Long:    long,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			keyID := args[0]
+
+			addr := ctx.GetFromAddress()
+
+			txf := tx.NewFactoryCLI(ctx, cmd.Flags())
+
+			if _, err := txf.Keybase().Key(keyID); err != nil {
+				return fmt.Errorf("keybase key: %w", err)
+			}
+
+			msgs := make([]sdk.Msg, 1)
+
+			msg, err := types.BuildMsgCreateSequencer(func(msg []byte) ([]byte, cryptotypes.PubKey, error) {
```

Comment:
`msg []byte` is actaully the payload to sign right? not to be confused with the `msg` created